### ISSUE TITLE
loss value

### ISSUE THREAD

**Issue by guoyanb** [hkmztrk/DeepDTA#5]
Hi, I am interested in  the work, and then I test this model on the DAVIS dataset. During the training, the loss value is very large. Have you ever met? What could be the reason?
![1](https://user-images.githubusercontent.com/39645139/54525899-29733a80-49b0-11e9-8894-8af985fa585a.png)

**Comment by hkmztrk** [comment]
Hello @guoyanb , thanks a lot for your interest in our work!
Those are the loss values when we did not perform log transformation on binding affinity values in Davis dataset. In the original dataset, for instance a sample with Kd=10000nM is represented as pkD=5. In that way it's easier to track loss value. 

I updated the repo and included a new argument --is_log, so that you'll be able to decide whether to apply log transformation or not:
if you are working on KIBA, --is_log 0 
if you are working on Davis, --is_log 1
